jcoin/josecoin.py

```diff
@@ -748,7 +748,7 @@ def main():
     """Main entrypoint."""
     loop = asyncio.get_event_loop()
 
-    server = app.create_server(host='0.0.0.0', port=8080)
+    server = app.create_server(host='0.0.0.0', port=jconfig.get("port", 8080))
```

Comment:
`jconfig` is a Python module, not a dict so it doesn't have a get function. If you want to make this config key optional you'll have to use `getattr(jconfig, 'port', 8080)` or an equivalent instead.